pkg/provider/provider_acceptance_test.go

```diff
@@ -168,6 +241,103 @@ provider "snowflake" {
 `, profile) + datasourceConfig()
 }
 
+func providerConfigWithRole(profile, role string) string {
```

Comment:
please add to our model builder generators readme that we should support model creation for the provider (if not added already)

Reply:
Done.